content/flux/v0.x/data-types/basic/duration.md

```diff
@@ -2,11 +2,129 @@
 title: Work with durations
 list_title: Duration
 description: >
-  ...
+  Learn how to work with duration data types in Flux.
 menu:
   flux_0_x:
     name: Duration
     parent: Basic types
 weight: 201
 flux/v0.x/tags: ["basic types", "data types"]
----
\ No newline at end of file
+related:
+  - /flux/v0.x/stdlib/universe/duration/
+---
+
+A **duration** type represents a length of time with nanosecond precision. 
+
+**Type name**: `duration`
+
+###### On this page:
+- [Duration syntax](#duration-syntax)
+- [Convert data types to durations](#convert-data-types-to-durations)
+- [Operate on durations](#operate-on-durations)
+
+## Duration syntax
+Duration literals contain integers and unit specifiers.
+Flux supports the following unit specifiers:
+
+- `ns`: nanosecond
+- `us`: microsecond
+- `ms`: millisecond
+- `s`: second
+- `m`: minute
+- `h`: hour
+- `d`: day
+- `w`: week
+- `mo`: calendar month
+- `y`: calendar year
+
+```js
+1ns // 1 nanosecond
+1us // 1 microsecond
+1ms // 1 millisecond
+1s  // 1 second
+1m  // 1 minute
+1h  // 1 hour
+1d  // 1 day
+1w  // 1 week
+1mo // 1 calendar month
+1y  // 1 calendar year
+
+3d12h4m25s // 3 days, 12 hours, 4 minutes, and 25 seconds
+```
+
+## Convert data types to durations
+Use the [`duration()` function](/flux/v0.x/stdlib/universe/duration/) to convert
+the following [basic types](/flux/v0.x/data-types/basic/) to durations:
+
+- **string**: must be formatted as a duration string.
+- **int**: parsed as nanoseconds and converted to a duration.
+- **uint**: parsed as nanoseconds and converted to a duration.
```

Comment:
Is duration string same as duration literal? and duration in lines 60 and 61 same as duration string?

Reply:
No a duration string would be `"1h30m"` (a duration formatted as string). A duration literal would be `1h30m` (the native literal syntax for durations).